packages/app-admin-ui/client/pages/List/index.js

```diff
@@ -70,7 +68,7 @@ export function ListLayout(props) {
         <HeaderInset>
           <FlexGroup align="center" justify="space-between">
             <PageTitle>{list.plural}</PageTitle>
-            {list.access.create ? listHeaderActions ? listHeaderActions() : <CreateItem /> : null}
+            {listHeaderActions ? listHeaderActions() : <CreateItem />}

```

Comment:
Do not want to remove all the header items which dev may have created, list create access check is moved to `<CreateItem/>` which will not render itself if there is no access for create.

we can leave this to hook author to decide whether they need action for create access or not.

Reply:
Good call to move it to create item.